# HG changeset patch
# User Andrew John Hughes <gnu.andrew@redhat.com>
# Date 1390503415 0
#      Thu Jan 23 18:56:55 2014 +0000
# Node ID 8796f8cdd6219d82827c5e3955e8e3142a81da38
# Parent  fa97228fa38479b74e413bc1c2d2aeef0057b6dc
Disable LCMS2 system library check when LCMS2 isn't required.

2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>

	* acinclude.m4:
	(IT_CHECK_FOR_LCMS): Depend on IT_ENABLE_LCMS2
	and only enable system LCMS by default if it's
	enabled.
	* configure.ac:
	Don't invoke IT_ENABLE_LCMS2 explicitly.

diff -r fa97228fa384 -r 8796f8cdd621 ChangeLog
--- a/ChangeLog	Thu Jan 23 18:46:47 2014 +0000
+++ b/ChangeLog	Thu Jan 23 18:56:55 2014 +0000
@@ -1,3 +1,12 @@
+2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>
+
+	* acinclude.m4:
+	(IT_CHECK_FOR_LCMS): Depend on IT_ENABLE_LCMS2
+	and only enable system LCMS by default if it's
+	enabled.
+	* configure.ac:
+	Don't invoke IT_ENABLE_LCMS2 explicitly.
+
 2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>
 
 	* patches/alternative_krb5_cache_fixup.patch:
diff -r fa97228fa384 -r 8796f8cdd621 acinclude.m4
--- a/acinclude.m4	Thu Jan 23 18:46:47 2014 +0000
+++ b/acinclude.m4	Thu Jan 23 18:56:55 2014 +0000
@@ -1537,6 +1537,7 @@
 
 AC_DEFUN_ONCE([IT_CHECK_FOR_LCMS],
 [
+  AC_REQUIRE([IT_ENABLE_LCMS2])
   AC_MSG_CHECKING([whether to use the system LCMS install])
   AC_ARG_ENABLE([system-lcms],
 	      [AS_HELP_STRING(--enable-system-lcms,use the system LCMS [[default=yes]])],
@@ -1544,7 +1545,7 @@
     ENABLE_SYSTEM_LCMS="${enableval}"
   ],
   [
-    if test x"${target_os}" = "xlinux-gnu"; then
+    if test x"${ENABLE_LCMS2}" = "xyes" -a x"${target_os}" = "xlinux-gnu" ; then
       ENABLE_SYSTEM_LCMS="yes" ;
     else
       ENABLE_SYSTEM_LCMS="no" ;
diff -r fa97228fa384 -r 8796f8cdd621 configure.ac
--- a/configure.ac	Thu Jan 23 18:46:47 2014 +0000
+++ b/configure.ac	Thu Jan 23 18:56:55 2014 +0000
@@ -296,10 +296,6 @@
 IT_WITH_PAX
 AC_CONFIG_FILES([pax-mark-vm], [chmod +x pax-mark-vm])
 
-IT_ENABLE_LCMS2
-
-IT_CHECK_FOR_KERBEROS
-
 dnl pkgconfig cannot be used to find these headers and libraries.
 AC_CHECK_HEADERS([cups/cups.h cups/ppd.h],[]
 	,[AC_MSG_ERROR([Could not find CUPS headers;
@@ -313,6 +309,7 @@
 IT_CHECK_FOR_PNG
 IT_CHECK_FOR_GIF
 IT_CHECK_FOR_LCMS
+IT_CHECK_FOR_KERBEROS
 AC_CONFIG_FILES([remove-intree-libraries.sh])
 
 dnl Check for Xproto headers and libraries.
